test/e2e/storage/csi_objects.go

```diff
@@ -140,6 +151,54 @@ func csiServiceAccount(
 	return ret
 }
 
+func csiPSPRoleBinding(client clientset.Interface,
+	config framework.VolumeTestConfig,
+	teardown bool,
+	sas []*v1.ServiceAccount) {
+	bindingString := "Binding"
+	if teardown {
+		bindingString = "Unbinding"
+	}
+	roleBindingClient := client.RbacV1beta1().RoleBindings(config.Namespace)
+	for _, sa := range sas {
+		By(fmt.Sprintf("%v cluster roles %v to the CSI service account %s", bindingString, podSecurityPolicyPrivilegedClusterRoleName, sa.Name))
+		binding := &rbacv1beta1.RoleBinding{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "psp-" + config.Namespace + "-" + string(uuid.NewUUID()),
+				Namespace: config.Namespace,
+			},
+			Subjects: []rbacv1beta1.Subject{
+				{
+					Kind:      rbacv1beta1.ServiceAccountKind,
+					Name:      sa.GetName(),
+					Namespace: sa.GetNamespace(),
+				},
+			},
+			RoleRef: rbacv1beta1.RoleRef{
+				Kind:     "ClusterRole",
+				Name:     podSecurityPolicyPrivilegedClusterRoleName,
+				APIGroup: "rbac.authorization.k8s.io",
+			},
+		}
+
+		roleBindingClient.Delete(binding.GetName(), &metav1.DeleteOptions{})
```

Comment:
Do we really need to clean up the namespace at the beginning of the test?  Every test case should run in a new namespace, so it should already be clean

Reply:
not necessary, but it doesn't hurt.